esi_leap/tests/api/controllers/v1/test_offer.py

```diff
@@ -27,6 +27,10 @@ def create_test_offer(context):
     return o
 
 
+def create_test_offer_data():
+    return {1: 'test', 2: 'offer'}
```

Comment:
Ah, we probably want to simulate actual test offer data more closely - something like:
```
{
   "resource_type": "test_node",
   "resource_uuid": "some-uuid",
   "start_time": "some_time_string"
}
```

Reply:
Ill add this in for offer. Someone made a ton of changes to contract so I figured I would make sure this works and then sift through the changes in the test to see where I should add it and such.